mathics/autoload/forms/StandardForm.m

```diff
@@ -12,7 +12,7 @@
    code. *)
 Attributes[CommonRadicalBox] = HoldAll;
 Attributes[RadBox] = HoldAll;
-CommonRadicalBox[expr_, form_] = RadBox[MakeBoxes[expr, form], 3];
+CommonRadicalBox[expr_, form_]:= RadBox[MakeBoxes[expr, form], 3];
```

Comment:
I know this may be wordy, but how about `CommonRadicalCubeBox` ? 

I made a mistake here in using the generic term "Radical" when what we are referring to is is a cube root radical.

"cube root" by itself is too generic because exponent 1/3 is a cube root as well, so what we want to convey is the radical form of a cube root.

Note that in the names here, we _are_ referring somewhat to the visual form, e.g. "radical" boxing as opposed to "power" boxing even though many times there is a box that surrounds the argument. 

The name though will influence _formatting_ later on.

Reply:
At this point, wordy is OK.